build.assets/install

```diff
@@ -15,5 +15,31 @@ cd $(dirname $0)
 mkdir -p $VARDIR $BINDIR
 cp -f teleport tctl tsh $BINDIR/ || exit 1
 
+#
+# What operating system is the user running?
+#
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=linux;;
+    Darwin*)    machine=darwin;;
+    CYGWIN*)    machine=cygwin;;
+    MINGW*)     machine=minGw;;
+    *)          machine="UNKNOWN:${unameOut}"
+esac
+
+#
+# What Teleport Version?
+# Sets version edition + version number e.g. Enterprisev5.1.0
+#
+teleportVersion="$(teleport version)"
+set -- $teleportVersion; version=$2$3
+
 echo "Teleport binaries have been copied to $BINDIR"
-echo "To configure the systemd service for Teleport take a look at examples/systemd/README.md"
+echo ""
+echo "Thanks for installing Teleport. Is it OK if we collect some metrics about your install?"
+echo ""
+echo "If you don't mind please run:"
+echo "# Update email if you would like to receive link to get swag"
+echo "$ curl -X POST https://9sjnq1g1j9.execute-api.us-west-2.amazonaws.com/Prod/usage -F OS=${machine} -F version=${version} -F email=alice@example.com"
```

Comment:
Note: This is a testing URL. We'll move to a shorter URL e.g. https://usage.teleport.dev

Reply:
Let's improve error messages:

```
curl -X POST https://9sjnq1g1j9.execute-api.us-west-2.amazonaws.com/Prod/usage
{"message": "Internal server error"}
```

Can you send a code in the same PR?

Reply:
Let's explain how we will use the data on submission:

```
curl -X POST https://9sjnq1g1j9.execute-api.us-west-2.amazonaws.com/Prod/usage -F email=alice@example.com
Thank you for submitting Teleport usage data. We will send you a meaningful newsletter and reach out to send some swag.
```

Reply:
I've fixed the error message if a user doesn't submit anything. I'll look at improving the message before submitting the survey.